doc/src/stdlib/math.md

```diff
@@ -232,7 +232,6 @@ Base.varm
 Base.middle
 Base.median
 Base.median!
-Base.midpoints
```

Comment:
this is in deprecated.jl but not actually deprecated, it still has tests even. we should move it to the 0.6 section, make it an actual deprecation, and remove the export and tests